be/src/olap/rowset/beta_rowset_reader.cpp

```diff
@@ -59,27 +59,45 @@ Status BetaRowsetReader::init(RowsetReaderContext* read_context) {
                                                  read_context->is_upper_keys_included->at(i));
         }
     }
+
+    bool can_reuse_schema = true;
     // delete_hanlder is always set, but it maybe not init, so that it will return empty conditions
     // or predicates when it is not inited.
     if (read_context->delete_handler != nullptr) {
```

Comment:
It seems that delete_handler is always not null whether delete condition exists.

Reply:
add check